### PR TITLE
:rocket: Release/v0.2.0

### LINKED ISSUE #147: Release v0.2.0
## Release

Release v0.2.0

## Checklist

- [x] Update metadata & clusterctl-settings.
- [x] Update version in e2e config.
- [x] Update docs (compatibility table; usage etc).
- [x] Run e2e tests on main branch 
- [x] Create release branch `release-X.Y`
- [x] Create tag `vX.Y.Z`.
- [x] Update the created draft release to include things like breaking changes or important notes.
- [x] Update the draft release notes with generated release notes from `sigs.k8s.io/kubebuilder-release-tools/notes`
- [x] Publish the release.

### PR DESCRIPTION
**Pull request by lubedacht**
**What is the purpose of this pull request/Why do we need it?**

Release PR for v0.2.0

**Issue #, if available:**

#147 

TODO
- [x] update version in e2e after #137 is merged

### CODE REVIEW

**Review by mcbenjemaa — CHANGES_REQUESTED** [review]
please also change clusterctl-settings, 
and if e2e got merged you also need to bump the version.

File: README.md [hunk 1]
```diff
@@ -39,9 +39,15 @@ If you need help with CAPIC, please visit the [#cluster-api-ionoscloud][slack] c
...
 
-### Cluster API Versions (TODO)
+### Cluster API Versions
 
-### Kubernetes Versions (TODO)
+This provider's versions are compatible with the following versions of Cluster API:
+
+|                        | Cluster API v1beta1 (v1.7) |
+|------------------------|:--------------------------:|
+| CAPIC v1alpha1 (v0.2)  |             ✓              |
```
Comment (mcbenjemaa) [inline README.md:48]: this should also include CAPI v1.6

Comment (lubedacht) [reply README.md:48]: As we haven't tested this with 1.6 - I would not include this at least not in this release. We can maybe run another test with 1.6 and add it to the matrix without a release

Comment (mcbenjemaa) [reply README.md:48]: As you like, 
But i just wanted the comparability table to look better with more support versions.

I know CAPI always works with +1/-1 minor version.

**Comment by mcbenjemaa** [comment]
Please update an item in release issue template, 

- Bump The version in E2e test config

**Review by mcbenjemaa — APPROVED** [review]
LGTM

**Pull request merged**